Added an entry for the accent-color CSS property

lukewarlow opened the pull request:
<!-- Please provide the following information to help us review this PR: -->

> What was wrong/why is this fix needed? (quick summary only)

This PR adds a page for the `accent-color` property from https://drafts.csswg.org/css-ui-4/#widget-accent.

PR to add the css formal definition: https://github.com/mdn/data/pull/490
PR to add the live example: https://github.com/mdn/interactive-examples/pull/1864
PR to add the browser compat data: https://github.com/mdn/browser-compat-data/pull/11470

This is my first time writing a full entry on MDN so apologies if I've missed a section that should be included.

lukewarlow committed:
Added an entry for the accent-color CSS property

Ryuno-Ki commented:
@rachelandrew Could you take a look?
I can't see any glaring issues from the markup, but cannot assess the subject itself.

teoli2003 commented:
Should we add a glossary entry for _accent color_? (Or a little bit more information).

I had to google it to understand it (maybe because I'm not a native speaker).

lukewarlow commented:
Perhaps including something like https://drafts.csswg.org/css-ui-4/#example-6760b48f would help make it clearer what it's referring to?

lukewarlow committed:
Added see compat table reference, and mentioned accent-color also shows in radio inputs

lukewarlow committed:
Make a start on a glossary page

lukewarlow committed:
Fix IDs

teoli2003 commented:
Yes, or a screenshot of one its use, though the live sample will do it, I think.

Maybe just a bit of text, maybe a paraphrase/simplificaation of this spec clause: "…a typically bright color that contrasts markedly with the more utilitarian background and foreground colors."

lukewarlow committed:
Link to glossary from main accent-color page

lukewarlow commented:
I've made a glossary page with a rough description. Right now it does basically just say what the spec says. Not sure on how best to reword it to be more friendly.

teoli2003 commented on files/en-us/web/css/accent-color/index.html:
```diff
@@ -0,0 +1,101 @@
+---
+<div>{{CSSRef}}{{SeeCompatTable}}</div>
+
+<p>The <strong><code>accent-color</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property sets the color of the elements <a href="/en-US/docs/Glossary/accent">accent</a>. An accent appears in elements such as {{HTMLElement("input")}} of <code><a href="/en-US/docs/Web/HTML/Element/input/checkbox">type="checkbox"</a></code>, or <code><a href="/en-US/docs/Web/HTML/Element/input/radio">type="r ...
```
{{Glossary("accent")}}

teoli2003 commented:
Cool! I will let @rachelandrew do the formal review as she can land stuff in mdn/data at the same time (or before).

lukewarlow committed:
Updated Glossary link

rachelandrew commented on files/en-us/web/css/accent-color/index.html:
```diff
@@ -0,0 +1,101 @@
+
+<p>The <strong><code>accent-color</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> property sets the color of the elements {{Glossary("accent")}}. An accent appears in elements such as {{HTMLElement("input")}} of <code><a href="/en-US/docs/Web/HTML/Element/input/checkbox">type="checkbox"</a></code>, or <code><a href="/en-US/docs/Web/HTML/Element/input/radio">type="radio"</a></code>.</p>
+
+<div>{{EmbedInteractiveExample("pages/css/accent-color.html")}}</div>
```
We wouldn't normally add an interactive example to something that requires flags to view it. So I suggest that you remove this until there are a couple of implementations.

lukewarlow replied:
Okay, I'll re-add in a follow up PR once it's shipped in Chrome.

rachelandrew reviewed (commented):
This looks fine, however as it's an experimental property we won't show an interactive example right now as it won't work for most people, which is confusing.

lukewarlow committed:
Remove live example

rachelandrew commented on files/en-us/glossary/accent/index.html:
```diff
@@ -0,0 +1,25 @@
+  - Input
+  - accent
+---
+<p><span class="seoSummary">An <strong>accent</strong> is a typically bright color that contrasts with the more utilitarian background and foreground colors within a color scheme.</span> These are present in the visual style of many platforms (though not all).</p>
```
```suggestion
<p>An <strong>accent</strong> is a typically bright color that contrasts with the more utilitarian background and foreground colors within a color scheme. These are present in the visual style of many platforms (though not all).</p>
```

rachelandrew committed:
Update files/en-us/glossary/accent/index.html

Removing the seoSummary as we are removing these from all docs in preparation for moving to markdown.

rachelandrew reviewed (approved):
Thank you!
